test/lib/meetup_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'test_helper'
+require_relative '../support/meetups/sample_api_responses'
+
+class MeetupTest < ActiveSupport::TestCase
+
+#The code is structured so /groups is always called before /events. Both responses must be stubbed.
+#Stub data is dependent on this urlname, do not change
+  setup do
+    Event.delete_all
+    Meetup.stubs(:get).with("/pro/operationcode/groups", options).returns(group_endpoint_response)
+    @urlname = "Operation-Code-Hampton-Roads"
+    Meetup.stubs(:get).with("/#{@urlname}/events", options).returns(event_endpoint_response)
+    @event1 = {:source_id=>"243869029",
```

Comment:
I added this because I was trying to update the events for testing with the event created by Faker which didn't work because Faker is class event and to update a db record I need a hash.